src/OrbitDB.js

```diff
@@ -244,13 +228,34 @@ class OrbitDB {
       accessController.add('write', this.key.getPublic('hex'))
```

Comment:
Would be nice if you can specify which keys to use here. If I'm not mistaken you can only determine the address on your own stores if you can't specify the public key(s).

Reply:
@oed Yeah, that's true; really I just kept this here so that the functionality of `create` remains the same. If you're trying to determine someone else's address, you'd need to pass in a write option, like `{ write: [...] }`. I think that should be sufficient?

Reply:
Oh, sorry. Didn't read the code well enough! Please disregard the previous message :)